src/test/java/org/carlspring/cloud/storage/s3fs/fileSystemProvider/NewAsynchronousFileChannelTest.java

```diff
@@ -97,29 +97,6 @@ void fileChannelSize()
         assertEquals(content.length(), size);
     }
 
-    @Test
-    void fileChannelAnotherSize()
-            throws IOException
```

Comment:
:+1: this is essentially the same test as `fileChannelSize` but with bigger `content` (bigger in terms of the first one is `content` and the second one is `content-more-large` lol :D)

:+1: for `fileChannelTruncate` as well (same thing)

Reply:
yeah, i've removed a lot of duplicates mostly. Tests from 13-17 years has a lot of those.